go/host/enclave/guardian.go

```diff
@@ -198,7 +198,6 @@ func (g *Guardian) mainLoop() {
 		// check enclave status on every loop (this will happen whenever we hit an error while trying to resolve a state,
 		// or after the monitoring interval if we are healthy)
 		g.checkEnclaveStatus()
-		g.logger.Trace("mainLoop - enclave status", "status", g.state.GetStatus())
```

Comment:
@BedrockSquirrel , I assume this log is somewhat useful

Reply:
Yeah very useful, and only trace level

Reply:
Same thing, tks for the find, putting it back in.
Seemed like the log was spinning before fixing the p2p thing.
<img width="960" alt="image" src="https://github.com/obscuronet/go-obscuro/assets/4471462/b8029b13-001c-47ae-8082-6a714595a83c">